test/system/700-play.bats

```diff
@@ -240,8 +240,14 @@ EOF
     TESTDIR=$PODMAN_TMPDIR/testdir
     mkdir -p $TESTDIR
     echo "$testYaml" | sed "s|TESTDIR|${TESTDIR}|g" > $PODMAN_TMPDIR/test.yaml
+
+    cgv1_preface=""
+    if is_cgroupsv1 && is_rootless; then
+      cgv1_preface="Resource limits.+ignored on.+"
+    fi
+
```

Comment:
@edsantiago what did I do wrong here?  [It still fails with this prefix in-place (log)](https://api.cirrus-ci.com/v1/artifact/task/5358200241782784/html/sys-podman-debian-12-rootless-host.log.html).

Reply:
Unfortunately, and counterintuitively, `assert` uses `expr`. You have to backslash the pluses... or just use dot-star.

Reply:
Ahh thanks, that would have taken me a long while to figure out.